traffic_ops/traffic_ops_golang/routes.go

```diff
@@ -56,6 +56,8 @@ func Routes(d ServerData) ([]Route, http.Handler, error) {
 		{1.2, http.MethodGet, "divisions-wip.json$", wrapHeaders(wrapAuthWithData(divisionsHandler(d.DB), d.Insecure, d.TOSecret, rd.PrivLevelStmt, DivisionsPrivLevel))},
 		{1.2, http.MethodGet, "hwinfo-wip$", wrapHeaders(wrapAuthWithData(hwInfoHandler(d.DB), d.Insecure, d.TOSecret, rd.PrivLevelStmt, HWInfoPrivLevel))},
 		{1.2, http.MethodGet, "hwinfo-wip.json$", wrapHeaders(wrapAuthWithData(hwInfoHandler(d.DB), d.Insecure, d.TOSecret, rd.PrivLevelStmt, HWInfoPrivLevel))},
+		{1.3, http.MethodGet, "deliveryservices/{xml-id}/urisignkeys$", wrapHeaders(wrapAuth(urisignkeysHandler(d.DB, d.Config), d.Insecure, d.TOSecret, rd.PrivLevelStmt, HWInfoPrivLevel))},
+		{1.3, http.MethodPost, "deliveryservices/{xml-id}/urisignkeys$", wrapHeaders(wrapAuthWithData(assignDeliveryServiceUriKeysKeysHandler(d.DB, d.Config), d.Insecure, d.TOSecret, rd.PrivLevelStmt, HWInfoPrivLevel))},
```

Comment:
Should a read-only user be able to read and update signing keys‽ I think key access should be a bit more restricted, but maybe we have it mostly open? Thoughts?

Reply:
I intend to add in tenancy checks to verify the user has access.  Was planning to do that next sprint.